Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -40,8 +89,108 @@ YUI.add('ez-usermodel-tests', function (Y) {
         tearDown: function () {
             this.model.destroy();
             delete this.model;
-        }
+        },
+
+        "Sync 'read' should load the content with CAPI": function () {
+            var m = this.model,
+                modelId = "/api/v2/ezp/model/mid",
+                callback = function () { };
+
+            Y.Mock.expect(this.capiMock, {
+                method: this.capiGetService,
+                returns: this.serviceMock
+            });
+            Y.Mock.expect(this.serviceMock, {
+                method: this.serviceLoad,
+                args: [
+                    modelId,
+                    Y.Mock.Value.Function
+                ],
+                run: Y.bind(function (id, callback) {
+                    callback(false, this.loadResponse);
+                }, this)
+            });
+
+            m.set('id', modelId);
+            m.sync('read', {
+                    api: this.capiMock
+                }, callback
+            );
+
+            Y.Mock.verify(this.capiMock);
+            Y.Mock.verify(this.serviceMock);
+        },
+
+        "Should not set avatar attribute with CAPI when user doesn't have avatar": function () {
+            var m = this.model,
+                modelId = "/api/v2/ezp/model/mid",
+                callback = function () { };
+
+            this.imageIdentifiers = [];
+
+            Y.Mock.expect(this.capiMock, {
+                method: this.capiGetService,
+                returns: this.serviceMock
+            });
+            Y.Mock.expect(this.serviceMock, {
+                method: this.serviceLoad,
+                args: [
+                    modelId,
+                    Y.Mock.Value.Function
+                ],
+                run: Y.bind(function (id, callback) {
+                    callback(false, this.loadResponse);
+                }, this)
+            });
 
+            m.set('id', modelId);
+            m.sync('read', {
+                    api: this.capiMock
+                }, callback
+            );
+
+            Y.Assert.isNull(this.model.get('avatar'), 'Should not set avatar when user doesn`t have one');
```

Comment:
this is not the right place to check that. This check should be done in the callback. Also to make sure the code is correctly executed, the avatar attribute in the model should be initialized to something else since `null` is also the default value of this attribute.